.build/tar/generate_tar.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+
+CONFIG_FILE=${CONFIG_FILE:-config-example.yaml}
```

Comment:
Not familiar with this script syntax here, but does the same example apply here? `CONFIG_FILE=<custom config file in the config directory> make build-tarball` runs this with a custom config file right?

Reply:
Yea sorry was going to add some comments for you.

This is `VARIABLE = ${VARIABLE:-DEFAULT_VALUE_IF_VARIABLE_NOT_SET}`

It does the same thing as before. I just moved it here since it only applies to this script and not everything in the makefile.

Reply:
perfect!